mopidy_spotify/search.py

```diff
@@ -55,11 +55,18 @@ def search(config, session, requests_session,
             'to at most 50.')
         search_count = 50
 
+    params = {
+        'q': sp_query,
+        'limit': search_count,
+        'type': ','.join(types)}
+
+    try:
+        params['market'] = session.user_country
+    except ValueError as exc:
```

Comment:
If that returned int is greater than 65535, then `utils.to_country` will raise ValueError when it tries to convert a number greater 255 to `chr`. That routine could do with another `& 0xff` in there.